fizz.go

```diff
@@ -360,6 +360,13 @@ func XCodeSample(cs *openapi.XCodeSample) func(*openapi.OperationInfo) {
 	}
 }
 
+// XInternal marks the operation as internal or external.
+func XInternal(isInternal bool) func(*openapi.OperationInfo) {
```

Comment:
Though we might not need an argument, since the default behavior is external (`internal: false`), so the option func could be used as simply as `XInternal()`.

Reply:
Agreed. I removed it now.